bokehjs/test/models/glyphs/rect.coffee

```diff
@@ -297,14 +297,14 @@ describe "Rect", ->
           expect(result3['1d'].indices).to.be.deep.equal([1])
 
         it "should work when axis is log", ->
-          data = {x: [60, 100, 140], y: [60, 100, 140]}
+          data = {x: [1, 10, 100, 1000], y: [1, 10, 100, 1000]}
```

Comment:
I'm unsure about this test. I understand the intent is to that `_hit_point` works by passing in vx/vy coordinates and checking that the method is able to calculate the glyph location in log coords. But in practice it's hard to know what the vx/vy coords should be or whether the test is passing for some unrelated reason (i.e. the glyph is massive and takes up the whole plot area).

It don't know if it's better to have a better structure for how to write these sorts of tests or to just replace it with a selenium test. Due to my changes, I don't think the tests covers anything meaningful.